handbook/engineering/security/secure-code-review.md

```diff
@@ -0,0 +1,41 @@
+# Requesting a security review
+
+Security code reviews differ from [peer code reviews](https://about.sourcegraph.com/handbook/engineering/code_reviews), as they are usually on large blocks of code, and after the fact. These reviews bring a second set of eyes, with dedicated focus to the problem.
```

Comment:
Should security review architecture before the fact?

Reply:
Agreed - and updated. I merged accidentally, but this (with clarification) is in main.